wechat/avatar.py

```diff
@@ -55,8 +55,13 @@ def get_avatar(self, username):
                     img_file = os.path.join(self.avt_dir, filename)
                     if os.path.exists(img_file):
                         return Image.open(img_file)
+                    elif os.path.exists(img_file + ".bm"):
+                        img = self.read_bm(img_file + ".bm")
+                        return Image.fromarray(img)
                     else:
-                        return None
+                        pos, size = self.query_index(filename)
+                        return self.read_img(pos, size)
```

Comment:
Why do you add this? query_index does not work when there is no avatar database.

Reply:
There're definitely situations need this, e.g. wechat updated from old version to new version. In such situation, both  avatar.index and png(.bm) are present, and unfortunately each one just cover a part.
There's a thing to concern here: If an avatar exists in both, we should prefer avatar.index first. But this little hack will in opposite.